Task2_2.py

```diff
@@ -0,0 +1,38 @@
+import string
+import random
+
+# creating list of dicts
+list_of_dicts = []
+# set number of dicts
+for k in range(random.randint(2, 10)):
+    # add random keys and values and number of key-value pairs
+    mydict = {chr(random.randint(ord('a'), ord('z'))): random.randint(0, 100) for i in range(random.randint(1, 100))}
```

Comment:
```suggestion
    mydict = {chr(random.randint(ord('a'), ord('z'))): random.randint(0, 100) for _ in range(random.randint(1, 100))}
```
Usually if you use for loop just to iterate through and don't use iterable value it's omitted by underscore.

Reply:
Moreover no mydict, yourdict, finaldict, prefinaldict etc.

Reply:
Got it.

Reply:
Actually no, they don't :) What sense should I see in finaldict? :)

Reply:
Well, the title says this version of the dictionary is final. But OK, I'll write something more meaningful.

Reply:
https://builtin.com/data-science/variable-names

Quite comprehensive tutorial :)